WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -159,7 +183,7 @@ class LoginPrologueViewController: LoginViewController {
     }
 
     private func googleTapped() {
-        guard WordPressAuthenticator.shared.configuration.enableUnifiedGoogle else {
+        guard configuration.enableUnifiedGoogle else {
```

Comment:
⚠️ Lines 156 and 165 refer to `WordPressAuthenticator.shared.configuration` as well. Should we also change those?

Reply:
Great catch! A bit more code in a closure, updated in: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/373/commits/bcbc13c763a460420ca56dd5cc48501ddadef101